packages/nodes-base/nodes/LoneScale/LoneScaleList.node.ts

```diff
@@ -0,0 +1,426 @@
+				name: 'type',
+				type: 'options',
+				required: true,
+				displayOptions: {
+					show: {
+						resource: ['item'],
+					},
+				},
+				options: [
+					{
+						name: 'Company',
+						value: 'COMPANY',
+						description: 'List of company',
+						action: 'Get company lists',
+					},
+					{
+						name: 'Contact',
+						value: 'PEOPLE',
+						description: 'List of contact',
+						action: 'Get contact lists',
+					},
+				],
+				default: 'PEOPLE',
+				description: 'Type of your list',
+				noDataExpression: true,
+			},
+			{
+				displayName: 'List Name or ID',
+				name: 'list',
+				type: 'options',
+				displayOptions: {
+					show: {
+						resource: ['item'],
+					},
+				},
+				typeOptions: {
+					loadOptionsMethod: 'getLists',
+					loadOptionsDependsOn: ['type'],
+				},
+				default: '',
+				description:
+					'Choose from the list, or specify an ID using an <a href="https://docs.n8n.io/code-examples/expressions/">expression</a>',
+				required: true,
+			},
+			{
+				displayName: 'First Name',
+				name: 'first_name',
+				type: 'string',
+				displayOptions: {
+					show: {
+						operation: ['add'],
+						resource: ['item'],
+						type: ['PEOPLE'],
+					},
+				},
+				default: '',
+				description: 'Contact first name',
+			},
+			{
+				displayName: 'Last Name',
+				name: 'last_name',
+				type: 'string',
+				displayOptions: {
+					show: {
+						operation: ['add'],
+						resource: ['item'],
+						type: ['PEOPLE'],
+					},
+				},
+				default: '',
+				description: 'Contact last name',
+			},
+			{
+				displayName: 'Full Name',
+				name: 'full_name',
+				type: 'string',
+				displayOptions: {
+					show: {
+						operation: ['add'],
+						resource: ['item'],
+						type: ['PEOPLE'],
+					},
+				},
+				default: '',
+				description: 'Contact full name',
+			},
+			{
+				displayName: 'Linkedin Url',
+				name: 'linkedin_url',
+				type: 'string',
+				displayOptions: {
+					show: {
+						operation: ['add'],
+						resource: ['item'],
+					},
+				},
+				default: '',
+				description: 'Contact Linkedin URL',
+			},
+			{
+				displayName: 'Company Name',
+				name: 'company_name',
+				type: 'string',
+				displayOptions: {
+					show: {
+						operation: ['add'],
+						resource: ['item'],
+					},
+				},
+				default: '',
+				description: 'Contact company name',
+			},
+			{
+				displayName: 'Current Position',
+				name: 'current_position',
+				type: 'string',
+				displayOptions: {
+					show: {
+						operation: ['add'],
+						resource: ['item'],
+						type: ['PEOPLE'],
+					},
+				},
+				default: '',
+				description: 'Contact current position',
+			},
+			{
+				displayName: 'Company Domain',
+				name: 'domain',
+				type: 'string',
+				displayOptions: {
+					show: {
+						operation: ['add'],
+						resource: ['item'],
+					},
+				},
+				default: '',
+				description: 'Contact company domain',
+			},
+			{
+				displayName: 'Contact Location',
```

Comment:
This is optional for both person and company, Should be moved under an Additional Items field